src/app/backend/resource/horizontalpodautoscaler/common.go

```diff
@@ -14,8 +14,7 @@
 
 package horizontalpodautoscaler
 
-// Simple mapping of an autoscaling.CrossVersionObjectReference
-type ScaleTargetRef struct {
-	Kind string `json:"kind"`
-	Name string `json:"name"`
-}
+import "k8s.io/client-go/pkg/apis/autoscaling/v1"
+
+// autoscaling.CrossVersionObjectReference
+type ScaleTargetRef v1.CrossVersionObjectReference
```

Comment:
If we decide to use `v1.CrossVersionObjectReference` we can do it without `ScaleTargetRef` alias. That means we can delete whole `common.go` file.